cogs/source/source.py

```diff
@@ -1,6 +1,8 @@
 from redbot.core import commands
+from redbot.core.bot import Red
 import tracemoepy
 from tracemoepy.errors import EmptyImage, EntityTooLarge, ServerError, TooManyRequests
+import asyncio
```

Comment:
I don't think we'll need asyncio here, but we can actually use logging to help us debug:
```suggestion
import logging
```